libheif/heif.h

```diff
@@ -161,73 +161,75 @@ enum heif_suberror_code
 
   heif_suberror_No_hvcC_box = 106,
 
-  heif_suberror_No_pitm_box = 107,
+   heif_suberror_No_vvcC_box = 107,
```

Comment:
Please make sure that all enum values in `heif.` do not change as that would break backwards compatibility.
Assign new numbers to new error codes.